.travis/build-deps.sh

```diff
@@ -63,23 +76,41 @@ build_openssl () {
 }
 
 # Enable ccache
-if [ "${TRAVIS_OS_NAME}" != "osx" ]; then
+if [ "${TRAVIS_OS_NAME}" != "osx" ] && [ -z "${CHOST+xxx}" ]; then
     # ccache not available on osx, see:
     # https://github.com/travis-ci/travis-ci/issues/5567
     mkdir -p "${HOME}/bin"
     ln -s "$(which ccache)" "${HOME}/bin/${CC}"
     PATH="${HOME}/bin:${PATH}"
 fi
 
-# Download and build crypto lib
-mkdir -p download-cache
-if [ "${SSLLIB}" = "openssl" ]; then
-    download_openssl
-    build_openssl
-elif [ "${SSLLIB}" = "mbedtls" ]; then
-    download_mbedtls
-    build_mbedtls
+if [ ! -z ${CHOST+xxx} ]; then
+      # cross build
+      echo "deb http://archive.ubuntu.com/ubuntu xenial main universe" | sudo tee -a /etc/apt/sources.list.d/xenial.list
+      echo "deb http://archive.ubuntu.com/ubuntu xenial main" | sudo tee -a /etc/apt/sources.list.d/xenial.list
+      sudo apt-get update
+      sudo apt-get -y install dpkg mingw-w64
+      download_openssl
+      if [ ! -f "download-cache/tap-windows-${TAP_WINDOWS_VERSION}.zip" ]; then
+         wget -P download-cache/ http://build.openvpn.net/downloads/releases/tap-windows-${TAP_WINDOWS_VERSION}.zip
+      fi
+      unzip download-cache/tap-windows-${TAP_WINDOWS_VERSION}.zip
+      #cp tap-windows-${TAP_WINDOWS_VERSION}/include/tap-windows.h ${PREFIX}/${PREFIX}/include
+      cp tap-windows-${TAP_WINDOWS_VERSION}/include/tap-windows.h include
```

Comment:
I couldn't get it running.

it would be nice if you will fork and try your idea